libp2p/Common.cpp

```diff
@@ -144,7 +144,7 @@ void DeadlineOps::reap()
         return;
 
     Guard l(x_timers);
-    std::vector<DeadlineOp>::iterator t = m_timers.begin();
+    auto t = m_timers.begin();
```

Comment:
Can this be const?

Reply:
No. 
It cannot be `const auto` - because of `t++` and `t = `.
It cannot be `.cbegin()` (const_iterator) because of `t->wait()` which cannot be a const method.